data/src/test/java/com/teamagam/gimelgimel/data/message/repository/cache/room/dao/SearchMessagesDaoTest.java

```diff
@@ -44,48 +58,50 @@ public void tearDown() {
   @Test
   public void TextMessageSearchTest() {
     //Arrange
-    ChatMessageEntity chatMessage = mock(ChatMessageEntity.class);
-    chatMessage.text = "static and benel are the best!";
+    ChatMessage chatMessage =
```

Comment:
Extract insertMessage method that gets a text and generates message and inserts it.
There're lots of dups here

Reply:
done